installer/install.iss

```diff
@@ -57,7 +57,7 @@ DisableDirPage=auto
 DefaultGroupName={#APP_NAME}
```

Comment:
Apparently I cannot comment on commits anymore, so I use the first commentable line of the diff instead.

InnoSetup's uninstaller does not use MSI under the hood (it can't).

Also, please move the GPL into the top-level directory in a separate, preparatory commit.

And I would not worry about custom installers. Anybody savvy enough to make a custom installer will be wise enough to expect the obvious when installing a non-custom Git on top.